pypeit/spectrographs/spectrograph.py

```diff
@@ -90,6 +91,7 @@ def __init__(self):
         self.rawdatasec_img = None
         self.oscansec_img = None
         self.slitmask = None
+        self.steps = None  # The reduction steps for this spectrograph
```

Comment:
I think there are a number of things in here that @profxj pointed out that we should talk about.  This is another one.  I'm not sure the reduction steps should be tied to the spectrograph class, but let's see what makes sense for KCWI.

Reply:
This is now done - I've removed `steps` and introduced `IFU` as a possible `PYPELINE`